<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>VGG16 TEST</string>
    </property>
    <widget class="QPushButton" name="q5_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Show Train Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q5_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>2. Show HyperParameter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q5_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Show Model Shortcut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q5_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>4. Show Accuracy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q5_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>5. Test</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
